### ISSUE TITLE
TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by Noyabronok** [bpxl-labs/react-component-hierarchy#8]
## Context

* *Node version*: 7
* *React-component-hierarchy version*: latest
* *Operating System and version*: OSX 10.11

## What are you trying to achieve or the steps to reproduce?

Got an error when trying to run rch on a `create-react-app` application
```
➜  src git:(master) ✗ rch ./App.js
```

## What were your results?

```
Unhandled rejection TypeError: Cannot read property 'length' of undefined
    at formatNodeToPrettyTree (/Users/theuser/.nvm/versions/node/v7.4.0/lib/node_modules/react-component-hierarchy/bin/rch.js:142:32)
    at done (/Users/theuser/.nvm/versions/node/v7.4.0/lib/node_modules/react-component-hierarchy/bin/rch.js:158:20)
    at readFile.then.catch (/Users/theuser/.nvm/versions/node/v7.4.0/lib/node_modules/react-component-hierarchy/bin/rch.js:186:11)
    at tryCatcher (/Users/theuser/.nvm/versions/node/v7.4.0/lib/node_modules/react-component-hierarchy/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/theuser/.nvm/versions/node/v7.4.0/lib/node_modules/react-component-hierarchy/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/theuser/.nvm/versions/node/v7.4.0/lib/node_modules/react-component-hierarchy/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/Users/theuser/.nvm/versions/node/v7.4.0/lib/node_modules/react-component-hierarchy/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/Users/theuser/.nvm/versions/node/v7.4.0/lib/node_modules/react-component-hierarchy/node_modules/bluebird/js/release/promise.js:689:18)
    at Async._drainQueue (/Users/theuser/.nvm/versions/node/v7.4.0/lib/node_modules/react-component-hierarchy/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/theuser/.nvm/versions/node/v7.4.0/lib/node_modules/react-component-hierarchy/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/Users/theuser/.nvm/versions/node/v7.4.0/lib/node_modules/react-component-hierarchy/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:651:20)
    at tryOnImmediate (timers.js:624:5)
    at processImmediate [as _immediateCallback] (timers.js:596:5)
```

## What did you expect?

For tree to print.  I am guessing this is due to import of `.css` files, since rch works when component doesn't import `.css` files

**Comment by piotrd** [comment]
I'm having the same issue with node version 6.10.1 and macOS 10.12.3. 

Have you maybe found any workaround so far, @Noyabronok?

**Comment by piotrd** [comment]
Ah, I just looked at PRs 🤦‍♂️ 

Why is your [PR](https://github.com/bpxl-labs/react-component-hierarchy/pull/9) not linked to this issue?

**Comment by Noyabronok** [comment]
it is now...if only the owner would accept and update npm

**Comment by brandonjpierce** [comment]
Hey guys, we are aware of the activity on this repo and are sorry for the lack of attention. We are currently tied up in some work but are actively trying to get to these PR's as soon as we can! Thank you for your patience and contributions :)

**Comment by survivorbat** [comment]
Having this issue as well, is there any news on this issue ?

**Comment by Surreal9** [comment]
This has been addressed in 1.1.0 and published to npm
